README.md

```diff
@@ -224,6 +225,10 @@ Type: `Boolean` or `Object`
 
 Default: `undefined` (do not write sourcemaps)
 
+##### other
```

Comment:
I generally agree with surfacing docs but I don't think people should be using these options so I'm good with them not being surfaced.